argilla/docs/tutorials/image_classification.ipynb

```diff
@@ -11,8 +11,8 @@
    "cell_type": "markdown",
```

Comment:
<p>I think we should remove "mps"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/argilla-io/argilla/pull/5448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='63'/>